go/exoservice/test-fixtures/send.go

```diff
@@ -15,12 +15,18 @@ type SendTextFixture struct {
 
 // GetMessageHandler returns a message hangler
 func (r *SendTextFixture) GetMessageHandler() exoservice.MessageHandlerMapping {
-	return exoservice.MessageHandlerMapping{
+	return exoservice.MessageHandlerMapping {
 		"ping": func(request exoservice.Request) {
 			err := request.Send(exorelay.MessageOptions{Name: "pong"})
 			if err != nil {
 				panic(fmt.Sprintf("Failed to send message: %v", err))
 			}
 		},
+    "ping it": func(request exoservice.Request) {
+			err := request.Send(exorelay.MessageOptions{Name: "pong it"})
+			if err != nil {
+				panic(fmt.Sprintf("Failed to send message: %v", err))
+			}
+		},
```

Comment:
Please format the go files with `[go fmt](https://blog.golang.org/go-fmt-your-code)`. It would be good to get an editor plugin for this as well. This isn't checked as part of the linting process at the moment (but is on git-town). I'll add an issue for myself to add it